src/typings/theme/typography.ts

```diff
@@ -0,0 +1,20 @@
+type baseTypograhy = {
+  fontSize: string | number
+  fontWeight?: number
+  lineHeight?: number
+  letteSpacing?: number
```

Comment:
spell error? `letterSpacing` and `baseTypography`
Pda: we should to decide if we are going to capitalize our types